index.html

```diff
@@ -1955,13 +1955,29 @@
 <div class="practice">
 <p>
   <span class="practicelab" id="principle-consent-withdraw">
-    It should be as easy for a user to check what consent they have given,
+  It should be as easy for a [=person=] to check what consent they have given,
 	to withdraw consent, or to opt out or object,
 	as to give consent.
   </span>
 </p>
 </div>
 
+A [=person=] providing consent does not imply consent from other [=people=] whose data may be
```

Comment:
I find the formulation a little confusing: is it the person who doesn't imply or the fact that they gave consent? I think the latter.

It's also a bit unclear what it means for someone else's data to be included or implicated? Maybe flip it around: "A person may share data some of which is also about other people (e.g. a picture with both that person and others). If that person consent to the processing of that data, this does not imply that those other people have consented as well."